sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/schemaregistry/encoder/avroencoder/_schema_registry_avro_encoder.py

```diff
@@ -75,7 +75,7 @@ class AvroEncoder(object):
     def __init__(
         self,
         *,
-        client: "SchemaRegistryClient",
+        client: "azure.schemaregistry.SchemaRegistryClient",
```

Comment:
oh why was it mad at this?

Reply:
It had to do with this error that was happening due an issue with sphinx and type annotations: https://github.com/sphinx-doc/sphinx/issues/4961#issuecomment-1543858623

Krista (👑) suggested that we do this here so sphinx can pick up the correct SchemaRegistryClient out of `azure.schemaregistry.SchemaRegistryClient, azure.schemaregistry._schema_registry_client.SchemaRegistryClient, azure.schemaregistry.aio.SchemaRegistryClient, azure.schemaregistry.aio._schema_registry_clientasync.SchemaRegistryClient` and it fixed the error.